tests/scripts/all.sh

```diff
@@ -153,7 +155,7 @@ while [ $# -gt 0 ]; do
 done
 
 if [ $FORCE -eq 1 ]; then
-    rm -rf yotta/module "$OUT_OF_SOURCE_DIR"
+    rm -rf yotta/module "$OUT_OF_SOURCE_DIR" mbed-cli/$MBED_APP
```

Comment:
I can see that it is necessary to import the mbed-os-example-tls project in a directory outside the mbed TLS source. However, I suggest to use the $OUT_OF_SOURCE_DIR instead of adding yet another directory to the project.